index.html

```diff
@@ -0,0 +1,35 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Trial Project</title>
+</head>
+<body>
+<header id="header">
+    <img id="header-img" src="#" alt="Landing logo" />
+    <nav id="nav-bar">
```

Comment:
Using id for block selectors containing multiple line is not advised according to HTML semantics. please use class instead